chains.config.js

```diff
@@ -9,7 +9,7 @@ const chains = [
     name: 'Gaia-X',
     networkId: GX_NETWORK_ID,
     nodeUri: 'https://rpc.gaiaxtestnet.oceanprotocol.com/',
-    providerUri: 'https://provider.gaiax.delta-dao.com/',
+    providerUri: 'http://ctd.ptw.tu-darmstadt.euprogigant.io',
```

Comment:
issue(blocking): Is there no HTTPS support?

Reply:
Not sure, I copied the link from the issue @3xz1 created

Reply:
Oh yeah of course it should use https. Just copied from EuProGigant.